src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -101,7 +131,7 @@ describe('HmctsGlobalHeaderComponent', () => {
     expect(component.navigate.emit).toHaveBeenCalled();
   });
 
-  it('should display find case right aligned', async () => {
+  it('should display find case right aligned', () => {
```

Comment:
@johnbenjamin-hmcts - given that this test involves doing a `subscribe()`, there may be asynchronous behaviour. If so, you might need to do what @ronaldmansveld did in some other tests: provide a `done` callback function, then call `done()` at the end of the test. Perhaps check with @ronaldmansveld whether this applies here.

Reply:
Thank you @Daniel-Lam - I have now implemented the changes.